packages/peregrine/lib/talons/Gallery/useAddToCartButton.js

```diff
@@ -6,7 +6,7 @@ import { useCartContext } from '../../context/cart';
 import operations from './addToCart.gql';
 
 /**
- * @param {Number} props.item.id - id of item
+ * @param {String} props.item.uid - uid of item
```

Comment:
It does still get the `id` though, right?

We're going to change `id` to `uid` coming up here pretty soon so I'm not actually requesting a change be made here.